src/Components/Projects/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { View } from 'react-native';
+import SectionHeader from './../SectionHeader';
+import Sponsors from './Sponsors';
+
+function Projects() {
+  return (
+    <View
+      style={{
+        width: '80%',
+        alignItems: 'left',
+        paddingLeft: 16,
+        paddingRight: 16,
+      }}
+    >
+      {/* <SectionHeader title="Partners" />
```

Comment:
@isabelcosta this section has been commented out.

Reply:
If this Sponsors section is commented, could you remove the 50+ binary files from the PR, or do you think it's beneficial to keep them in the git history? @annabauza